source/code-snippets/usage-examples/count.js

```diff
@@ -15,16 +21,17 @@ async function run() {
     const estimate = await movies.estimatedDocumentCount();
     console.log(`Estimated number of documents in the movies collection: ${estimate}`);
 
-    // Query for movies from Canada.
+    // Query for movies where the countries field includes "Canada".
```

Comment:
Suggestion:
Since this only creates an object that represents a query, I would omit this comment. If I'm interpreting this correctly, this is a narrative comment that might refer to a larger block of commands. However, when those blocks contain comments that apply just to the line or lines immediately afterwards, it's difficult to discern what the comments apply to.